src/OGL3X/GLSLCombiner_ogl3x.cpp

```diff
@@ -142,18 +142,25 @@ void InitZlutTexture()
 	glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MIN_FILTER, GL_NEAREST);
 	glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_WRAP_S, GL_CLAMP_TO_EDGE);
 	glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_WRAP_T, GL_CLAMP_TO_EDGE);
+
+#ifndef GLESX
 	glTexImage2D(GL_TEXTURE_2D, 0, fboFormats.lutInternalFormat,
 		512, 512, 0, fboFormats.lutFormat, fboFormats.lutType,
 		zLUT);
-	glBindImageTexture(ZlutImageUnit, g_zlut_tex, 0, GL_FALSE, 0, GL_READ_ONLY, fboFormats.lutInternalFormat);
+#else
+	glTexStorage2D(	GL_TEXTURE_2D, 1, fboFormats.lutInternalFormat, 512, 512);
+#endif
```

Comment:
glBindImageTexture requires an immutable texture object. See https://www.khronos.org/opengles/sdk/docs/man31/html/glBindImageTexture.xhtml

glTexStorage2D doesn't create immutable texture objects but glTexStorage2D does.

I think you did make me realize a mistake. glBindImageTexture takes texture data as an argument, glTexStorage2D doesn't. I need to find how to set texture data after.